python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-adaptive-virtual-topology.j2

```diff
@@ -53,6 +53,31 @@ router adaptive-virtual-topology
 {%         if profile.load_balance_policy is arista.avd.defined %}
       path-selection load-balance {{ profile.load_balance_policy }}
 {%         endif %}
+{%         if profile.metric_order is arista.avd.defined %}
+      metric order {{ profile.metric_order }}
+{%         endif %}
+{%         if profile.outlier_elimination.enabled is arista.avd.defined(true) %}
+      path-selection outlier elimination
+{%         elif profile.outlier_elimination.enabled is arista.avd.defined(false) %}
+      path-selection outlier elimination disabled
+{%         endif %}
+{%         if profile.outlier_elimination.threshold.latency is arista.avd.defined or
+               profile.outlier_elimination.threshold.jitter is arista.avd.defined or
+               profile.outlier_elimination.threshold.loss_rate is arista.avd.defined or
```

Comment:
We can eliminate these checks.

Reply:
still kept the if profile.outlier_elimination.threshold is arista.avd.defined